src/views/hostnetworkconfiguration/components/TopologySidebar/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { IoK8sApiCoreV1Node } from '@kubevirt-ui/kubevirt-api/kubernetes/models';
+import { V1beta1NodeNetworkConfigurationPolicy } from '@types';
+import { isEmpty } from '@utils/helpers';
+
+export const isPolicyAppliedInNode = (
+  policy: V1beta1NodeNetworkConfigurationPolicy,
+  node: IoK8sApiCoreV1Node,
+) => {
+  if (isEmpty(policy.spec.nodeSelector)) return true;
```

Comment:
Are the `policy` object and each of these properties guaranteed to exist?

Reply:
yes. At least the policy spec. nodeSelector can be omitted but we are testing this before going deeper